apivore.gemspec

```diff
@@ -11,6 +11,7 @@ Gem::Specification.new do |s|
   s.files       = ['lib/apivore.rb', 'lib/apivore/*', 'data/swagger_2.0_schema.json']
   s.homepage    = 'http://github.com/westfieldlabs/apivore'
   s.add_runtime_dependency 'json-schema', '~> 2.5.1'
+  s.add_runtime_dependency 'rspec', '~> 3'
```

Comment:
This is going to conflict with my other PR https://github.com/westfieldlabs/apivore/pull/47 that updates the gemspec to be inline with the latest v1.0.0 code. I kept the two changes separate, but they will need to be merged now.

If this looks good, I'll merge these changes into PR 47